example/views/index.hbs

```diff
@@ -0,0 +1,10 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Welcome</title>
+</head>
+<body>
+  Welcome {{user}}
+</body>
+</html>
```

Comment:
Please add a newline to the end of this file, or [it won't be universally acknowledged as a text file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file/18789#18789).